qa/L0_backend_python/python_test.py

```diff
@@ -53,36 +56,40 @@ def _infer_help(self, model_name, shape, data_type):
             output0 = result.as_numpy('OUTPUT0')
             self.assertTrue(np.all(input_data_0 == output0))
```

Comment:
May not be worth it for this PR, but heads up for the future that Numpy has its own testing class that returns detailed messages for assertions like this one. Example for this case: https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_array_equal.html

Reply:
Filed a ticket.